### DISCUSSION TITLE
Does Concourse have stewardship since the Broadcom  buyout?

### DISCUSSION THREAD

**Discussion by norman-abramovitz** [concourse/concourse#9014] [Help & Support] [answered]
Does Concourse have active stewardship? If not, is their effort afoot to bring it back to life?

**Comment by taylorsilva** [comment, marked as answer]
Hello, there is effort afoot, by yours truly! I am not part of Broadcom, I am a former Pivotal/VMware employee who worked on Concourse ([contributors page](https://github.com/concourse/concourse/graphs/contributors))

You can see progress here: https://github.com/concourse/concourse/issues/8938

This comment from that issue is the tldr: https://github.com/concourse/concourse/issues/8938#issuecomment-2354379852

**Reply by norman-abramovitz** [reply]
Thank you @taylorsilva .
